rllib/offline/estimators/doubly_robust.py

```diff
@@ -0,0 +1,66 @@
+from ray.rllib.offline.estimators.off_policy_estimator import OffPolicyEstimate
+from ray.rllib.offline.estimators.direct_method import DirectMethod, k_fold_cv
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.typing import SampleBatchType
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.utils.numpy import convert_to_numpy
+import numpy as np
+
+
+class DoublyRobust(DirectMethod):
+    """The Doubly Robust (DR) estimator with Q-Reg Q-function.
+
+    DR estimator described in https://arxiv.org/pdf/1511.03722.pdf,
+    Q-Reg DR in https://arxiv.org/pdf/1911.06854.pdf"""
+
+    @override(DirectMethod)
+    def estimate(self, batch: SampleBatchType) -> OffPolicyEstimate:
+        self.check_can_estimate_for(batch)
+        estimates = []
+        # Split data into train and test using k-fold cross validation
+        for train_episodes, test_episodes in k_fold_cv(batch, self.k):
+            # Reinitialize model
+            self.model.reset()
+
+            # Train Q-function
+            if train_episodes:
+                train_batch = train_episodes[0].concat_samples(train_episodes)
```

Comment:
someone who is not super familiar with SampleBatch would think train_episodes[0] here is duplicated twice in the concatenated batch.
but actually concat_samples() is a static method ...
can you simply use SampleBatch.concat_samples() instead?